### PR TITLE
table designer editor icon

### PR DESCRIPTION
**Pull request by alanrenmsft**
light
image

dark
image

### CODE REVIEW

File: src/sql/workbench/browser/editor/tableDesigner/tableDesignerInput.ts [hunk 1]
```diff
@@ -52,8 +53,11 @@ export class TableDesignerInput extends EditorInput {
 		return TableDesignerInput.ID;
 	}
 
-	get resource(): URI {
-		return undefined;
+	public get resource(): URI {
+		return URI.from({
+			scheme: Schemas.tmp,
```
Comment (Charles-Gagnon) [inline src/sql/workbench/browser/editor/tableDesigner/tableDesignerInput.ts:58]: tmp? Where's that from?

**Review by Charles-Gagnon — APPROVED** [review]
Did you get this from the designers? Might be good to have a little edit pencil in it too (not sure if that'll get too crowded though)

File: src/sql/workbench/browser/editor/tableDesigner/tableDesignerInput.ts (hunk 1 above)
Comment (alanrenmsft) [reply src/sql/workbench/browser/editor/tableDesigner/tableDesignerInput.ts:58]: https://github.com/microsoft/azuredatastudio/blob/cb7b7da0a4bb2b45f0b6092e42fd1ca385938f09/src/vs/base/common/network.ts#L93

image

looks to be the best fit for the table designer scenario.

Comment (Charles-Gagnon) [reply src/sql/workbench/browser/editor/tableDesigner/tableDesignerInput.ts:58]: What about `vscodeCustomEditor`?

**Comment by alanrenmsft** [comment]
> Did you get this from the designers? Might be good to have a little edit pencil in it too (not sure if that'll get too crowded though)

this is the icon we used in OE for table. I can ask the designer to see whether they can come up with something.

File: src/sql/workbench/browser/editor/tableDesigner/tableDesignerInput.ts (hunk 1 above)
Comment (Charles-Gagnon) [reply src/sql/workbench/browser/editor/tableDesigner/tableDesignerInput.ts:58]: Actually I'd be wary of using a schema here and having other side effects...for example in https://github.com/microsoft/azuredatastudio/blob/main/src/vs/workbench/contrib/customEditor/browser/customEditorInputFactory.ts#L126

It'd probably be safest just to make our own set of Schemas and just use that

**Pull request merged**